framework/format/format.h

```diff
@@ -260,7 +260,7 @@ struct AnnotationHeader
     BlockHeader    block_header;
     AnnotationType annotation_type;
     uint32_t       label_length;
-    uint32_t       data_length;
+    uint64_t       data_length;
```

Comment:
@bradgrantham-lunarg  I kept the label length 32 bit and just boosted the data one.